Volontulo/app/src/main/java/com/stxnext/volontulo/MainActivity.java

```diff
@@ -0,0 +1,38 @@
+package com.stxnext.volontulo;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.widget.Toast;
+
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        ButterKnife.bind(this);
```

Comment:
Maybe this line go to some base activity that be automatically in all activities injected